doc/api/deprecations.md

```diff
@@ -1031,6 +1031,17 @@ With the current crypto API, having `Cipher.setAuthTag()` and
 when called. They have never been documented and will be removed in a future
 release.
 
+<a id="DEP00XX"></a>
+### DEP00XX: crypto.prng(), crypto.pseudoRandomBytes(), crypto.rng()
+
+Type: Runtime
+
+In recent versions of Node.js, there is no difference between
+[`crypto.randomBytes()`][] and `crypto.pseudoRandomBytes()`. The latter has been
```

Comment:
Nit: `has been` -> `is` for consistency with the rest of the deprecation messages (with one exception that should also be changed but not in this PR).